src/components/ConfirmStatusModal/ConfirmStatusModal.js

```diff
@@ -71,7 +71,7 @@ class ConfirmStatusModal extends React.Component {
           </Button>}
       </ModalFooter>
     );
-    const messageParts = message.map(m => <p>{m}</p>);
+    const messageParts = message.map((m, i) => <p key={i}>{m}</p>);
```

Comment:
Agree, `key` attribute must be used! 
But this is not the solution, `index` can not become the `key` value. Or this is the same as not to use `key` at all?
**From the [react doc](https://reactjs.org/docs/lists-and-keys.html)**: "If you choose not to assign an explicit key to list items then React will default to using indexes as keys."
 We need to provide some pattern for generating unique `key`, because in larger half of cases in our app we have only `index`, but any index-based `key` will be failed. Maybe we should try to use `faker.random.uuid()` in code?
https://stackoverflow.com/questions/46735483/error-do-not-use-array-index-in-keys